### PR TITLE
feat: Addition of npm cli cheat sheet

### PR DESCRIPTION
**Pull request by lholmquist**

### CODE REVIEW

File: npm-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+|`--preid=<prerelease-id>`
+| Use a custom prerelease id tag.  ex: `npm version prelease --preid=beta` example output: `1.1.0-beta.0`
+
+|`from-git`
+| Use the latest git tag as the version
+|===
+
+`npm shrinkwrap`:: Lock down dependency versions for publication
+
+`npm publish`:: Publish a module
```
Comment (richardlau) [inline npm-cli.adoc:104]: May be useful to mention the `--dry-run` option to see what would be published without actually publishing.

Comment (lholmquist) [reply npm-cli.adoc:104]: Yea,  good call on that.  i'll add it

File: npm-cli.adoc [hunk 2]
```diff
@@ -64,7 +64,7 @@ This cheat sheet covers the most common and helpful commands and flags for the n
 
 `npm login`:: Login to the registry
 
-`npm version [options]`:: Bump the version of a module
+`npm version [options] [--dry-run]`:: Bump the version of a module.  Use the `--dry-run` flag to get a preview without updating
```
Comment (richardlau) [inline npm-cli.adoc:67]: `--dry-run` should be on `npm publish`

**Review by richardlau — CHANGES_REQUESTED** [review]

File: npm-cli.adoc (hunk 2 above)
Comment (lholmquist) [reply npm-cli.adoc]: :facepalm:  of course it does.    my use of `standard-version` started to blend with npm usage

File: npm-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+This cheat sheet covers the most common and helpful commands and flags for the npm CLI.
+
+== Basic Usage
+
+`-v, --version`:: Prints the current version of npm you are using
+`<command> -h, --help`:: quick help on the specified command
+`-l`:: display full usage info
+
+`npx [command]`:: Run an arbitrary command from an npm module (either one installed locally, or fetched remotely)
```
Comment (richardlau) [inline npm-cli.adoc:11]: This still works in npm 7 but I believe the npm team are trying to steer towards `npm exec` which is npm 7 only.

**Review by richardlau — APPROVED** [review]

**Pull request merged**